apl/scalar-functions/mathematical-functions/max-of.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: max_of
+description: 'This page explains how to use the max_of function in APL.'
+---
+
+Use the `max_of` function in APL (Axiom Processing Language) to return the maximum value from a list of scalar expressions. You can use it when you need to compute the maximum of a fixed set of values within each row, rather than across rows like with aggregation functions. It is especially useful when the values you want to compare come from different columns or are dynamically calculated within the same row.
```

Comment:
Perhaps a link to the aggregation functions. Seems like people may go from one to the other.